### PR TITLE
Add remaining template text-only prebuilts

### PR DESCRIPTION
**Pull request by dagood**
```
Microsoft.DotNet.Common.ItemTemplates/1.0.2-beta3
Microsoft.DotNet.Common.ItemTemplates/2.0.0-preview8.19373.1
Microsoft.DotNet.Common.ItemTemplates/3.1.8
Microsoft.DotNet.Web.ItemTemplates/5.0.0
Microsoft.DotNet.Web.ProjectTemplates.5.0/5.0.0
Microsoft.DotNet.Web.Spa.ProjectTemplates.5.0/5.0.0
```

### CODE REVIEW

File: src/textOnlyPackages/src/microsoft.dotnet.web.itemtemplates/5.0.0/.nupkg.metadata [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (dseefeld) [inline src/textOnlyPackages/src/microsoft.dotnet.web.itemtemplates/5.0.0/.nupkg.metadata:1]: I don't think this file should be included.

File: src/textOnlyPackages/src/microsoft.dotnet.web.itemtemplates/5.0.0/microsoft.dotnet.web.itemtemplates.5.0.0.nupkg.sha512 [hunk 2]
```diff
@@ -0,0 +1 @@
+AJnFCMlcVXJAByLoht4nGQpWwN5w6PaqX+3GYJrueCEQnbu3HmVBwgLGl3UU/4IbIs8hD1el43YBnTWk+pctUg==
```
Comment (dseefeld) [inline src/textOnlyPackages/src/microsoft.dotnet.web.itemtemplates/5.0.0/microsoft.dotnet.web.itemtemplates.5.0.0.nupkg.sha512:1]: I don't think this file should be included.

File: src/textOnlyPackages/src/microsoft.dotnet.web.itemtemplates/5.0.0/.nupkg.metadata (hunk 1 above)
Comment (dagood) [reply src/textOnlyPackages/src/microsoft.dotnet.web.itemtemplates/5.0.0/.nupkg.metadata:1]: Huh... not sure how I missed this one. It's also missing the csproj.

**Review by dseefeld — APPROVED** [review]
I don't know how to leave a comment on a binary file, but there's a .signature.p7s file included that shouldn't be as well.

Otherwise, LGTM

File: src/textOnlyPackages/src/microsoft.dotnet.web.itemtemplates/5.0.0/.nupkg.metadata (hunk 1 above)
Comment (dagood) [reply src/textOnlyPackages/src/microsoft.dotnet.web.itemtemplates/5.0.0/.nupkg.metadata]: VS Code is acting a little oddly with detecting file system changes right now. The manual refresh button wasn't quite working either--had to do a window reload. Should be good now though.

**Pull request merged**